test/README.md

```diff
@@ -117,7 +117,75 @@ environment.
 
    For more info see [kubectl-gather](https://github.com/nirs/kubectl-gather)
 
-### Testing that drenv is healthy
+## Setup on macOS
+
+1. Install the [Homebrew package manager](https://brew.sh/)
+
+1. Install required packages
+
+   ```
+   brew install make go kubectl kustomize helm velero virtctl minio-mc argocd
```

Comment:
make should not be needed - make it part xcode tools installed by other components.

Installing make with brew shows:

```
==> Fetching make
==> Downloading https://ghcr.io/v2/homebrew/core/make/blobs/sha256:94377dc5a364da305c75fd7aa923a42897993de9edd1eb074428e13c3f2aaf93
############################################################################################################################################################### 100.0%
==> Pouring make--4.4.1.arm64_sonoma.bottle.1.tar.gz
==> Caveats
GNU "make" has been installed as "gmake".
If you need to use it as "make", you can add a "gnubin" directory
to your PATH from your bashrc like:

    PATH="/opt/homebrew/opt/make/libexec/gnubin:$PATH"
```